includes/payment-methods/class-wc-stripe-upe-payment-method.php

```
@@ -121,20 +121,28 @@ abstract class WC_Stripe_UPE_Payment_Method extends WC_Payment_Gateway {
 	 */
 	protected $oc_enabled;
 
+	/**
+	 * Whether this payment method is the first available payment method.
+	 *
+	 * @var bool
+	 */
+	protected $is_first_available_payment_method;
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $is_first_available_payment_method. Keep variable name length under 20. (undefined)

(LongVariable)